docs/moderators.md

```diff
@@ -94,3 +94,24 @@ Adding protected rooms on the fly is as easy as `!mjolnir rooms add <room alias>
 which are protected with `!mjolnir rooms`, and remove a room with `!mjolnir rooms remove <room alias>`. Note
 that rooms which are listed in the config may be protected again when the bot restarts - to remove these rooms
 permanently from protection, remove them from the config.
+
+## Trusted Reporters
+
+Mjolnir has an (optional) system in which it will poll Synapse for new reports, and when it sees sufficient
+(configurable limit) amounts of reports from trusted (configurable list) users on an given report, it will
+take a configurable action, such as redacting the message.
+
+Prerequisites:
+* `pollReport: true` in Mjolnir config file
+* retart Mjolnir
+* `!mjolnir enable TrustedReporters`
+* `!mjolnir config add TrustedReporters.mxids @trusteduser:example.com`
+* `!mjolnir config set TrustedReporters.alertThreshold 3`
+
+TrustedReporters supports 3 different thresholds; `alertThreshold`, `redactThreshold`, and `banThreshold`.
```

Comment:
Are there docs explaining what alert, redact and ban thresholds do and when they are triggered?

Reply:
the "when" is meant to be implied by the rest of the text; you configure the thresholds and the only one defined by default is `alert`